### PR TITLE
clarify Snowflake account location

### PR DESCRIPTION
**Pull request by lcochran400**
Because the url of Snowflake, where the account information can be easily found, changes based on which version of Snowflake the user is on (Classic Console vs AppUI/Snowsite), it can be confusing to identify the location of the account information. I added a distinction for each console to provide more clarity.

## Prerelease docs
If this change is related to functionality in a prerelease version of dbt (delete if not applicable):
- [ ] I've added versioning components, as described in ["Versioning Docs"](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/versioningdocs.md)
- [ ] I've added a note to the prerelease version's [Migration Guide](https://github.com/dbt-labs/docs.getdbt.com/tree/current/website/docs/guides/migration/versions)

## Checklist
If you added new pages (delete if not applicable):
- [ ] The page has been added to `website/sidebars.js`
- [ ] The new page has a unique filename

If you removed existing pages (delete if not applicable):
- [ ] The page has been removed from `website/sidebars.js`
- [ ] An entry has been added to `_redirects`

### CODE REVIEW

**Review by coapacetic — APPROVED** [review]
LGTM 🚀

File: website/docs/guides/getting-started/getting-set-up/setting-up-snowflake.md [hunk 1]
```diff
@@ -242,7 +242,7 @@ Now let's formally set up the connection between dbt Cloud and Snowflake.
...
 2. Enter the following information under Snowflake settings.
-    * **Account:** Your account is going to be the url of your Snowflake trial account up to the period before snowflakecomputing.com. So, if this is the full url of my trial account: `oq65696.west-us-2.azure.snowflakecomputing.com` then the account will be: `oq65696.west-us-2.azure`. You can read more about Snowflake account identifiers [here](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html).
+    * **Account:** Your account is located in the url of your Snowflake trial account, excluding the `snowflakecomputing.com`. The location of your account information will vary based on which version of Snowflake you are using. For Snowflake's Classic Console, your url may look like this: `oq65696.west-us-2.azure.snowflakecomputing.com`; if you're using the AppUI, or Snowsite, then the url may look like this: `snowflakecomputing.com/west-us-2.azure/oq65696`. In both of these instances, your account will be: `oq65696.west-us-2.azure`. You can read more about Snowflake account identifiers [here](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html).
```
Comment (b-per) [inline website/docs/guides/getting-started/getting-set-up/setting-up-snowflake.md]: It is actually Snowsight, not Snowsite

Comment (runleonarun) [inline website/docs/guides/getting-started/getting-set-up/setting-up-snowflake.md]:
```suggestion
    * **Account:** Find your account by using the Snowflake trial account URL and removing `snowflakecomputing.com`. The order of your account information will vary by Snowflake version. For example, Snowflake's Classic console URL might look like: `oq65696.west-us-2.azure.snowflakecomputing.com`. The AppUI or Snowsight URL might look more like: `snowflakecomputing.com/west-us-2.azure/oq65696`. In both examples, your account will be: `oq65696.west-us-2.azure`. For more information, see "[Account Identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier.html)" in the Snowflake documentation.
```

**Review by runleonarun — APPROVED** [review]
Thank you for opening this PR @lcochran-dbt! I slightly reworded to make the sentence active. We've been trying to standardize a few things like references to more info. Not everything has made it to the [style guide](https://github.com/dbt-labs/product-docs-team/blob/main/content-style-guide.md) just yet.

**Pull request merged**